data/styles/clear/include/Icons.mapcss

```diff
@@ -1400,6 +1400,9 @@ area|z15-[amenity=place_of_worship][religion=jewish]
 node|z15-[amenity=place_of_worship][religion=taoist],
 area|z15-[amenity=place_of_worship][religion=taoist]
 {icon-image: taoist-m.svg;}
+node|z15-[amenity=place_of_worship][religion=christian][denomination=mormon],
+area|z15-[amenity=place_of_worship][religion=christian][denomination=mormon]
```

Comment:
nit: better to add a trailing comma always, it avoids some diff noise if we add more types in the block later (probably not in this exact case, but good to follow a uniform style)

Reply:
Fixed